tests/foreman/ui/test_computeprofiles.py

```diff
@@ -42,13 +42,19 @@ def test_positive_end_to_end(session, module_loc, module_org):
     ).create()
     with session:
         session.computeprofile.create({'name': name})
-        assert session.computeprofile.search(name)[0]['Name'] == name
+
```

Comment:
I have a query ... why do you not want to check it with UI ? if it search in API way then it's missing the search check on UI ?

Are we covering the same functionality in other UI test? (I am thinking if incase any regression would be around search then that would not caught with this test around same entity)